gtw.yml

```diff
@@ -0,0 +1,20 @@
+amplifier-gateway:
+    image: appcelerator/amplifier-gateway:local
+    networks:
+      default:
+        aliases:
+          - amplifier-gateway
+    labels:
+      io.amp.role: "infrastructure"
+    environment:
+      - "SERVICE_PORTS=80"
+      - "VIRTUAL_HOST=https://gw.*"
+    ports:
+      - "8080:80"
+    deploy:
```

Comment:
we should add a placement constraint to restrict it on the worker nodes (see other stack files)